### ISSUE TITLE
What is "Microsoft Viva Insights *premium*"? This term is used nowhere else.

### ISSUE THREAD

**Issue by datto-cschulze** [MicrosoftDocs/Viva#479]
I've searched the internet but cannot find any info what Microsoft Viva Insights *premium* actually is and how to order it.

FYI I am using an Office 365 E3 license. There is no Teamwork habits card or tab on https://insights.viva.office.com/ available.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 65cba4ff-5bd9-f9a5-d68c-c3e5b2c16188
* Version Independent ID: 65cba4ff-5bd9-f9a5-d68c-c3e5b2c16188
* Content: [Teamwork habits in Viva Insights](https://learn.microsoft.com/en-us/viva/insights/org-team-insights/teamwork-habits#disable-or-enable-teamwork-habits)
* Content Source: [Viva/insights/org-team-insights/teamwork-habits.md](https://github.com/MicrosoftDocs/OfficeDocs-WorkplaceAnalytics/blob/public/Viva/insights/org-team-insights/teamwork-habits.md)
* Service: **viva**
* Sub-service: **viva-insights**
* GitHub Login: @lilyolason
* Microsoft Alias: **v-lilyolason**

**Comment by scanum** [comment]
Thank you for submitting feedback.
I think the best way forward is if you open a service ticket in your tenant so this can get resolved ASAP. Based on the outcome let me know if it is something that can be called out in the docs.
Please follow this link to contact support for business products:https://docs.microsoft.com/office365/admin/contact-support-for-business-products
Please keep us posted here on the resolution so we can feed whatever information you discover into the content.

**Comment by datto-cschulze** [comment]
@scanum Sorry, my first post was perhaps not so good. Let me rephrase my question: Are you sure the product is called "Microsoft Viva Insights _premium_"? I slightly doubt it, because this term is nowhere to be found.
